src/run/poll_results.rs

```diff
@@ -98,19 +99,19 @@ pub async fn poll_results(
     end_group!();
 
     if !response.run.results.is_empty() {
-        start_group!("Benchmarks");
+        start_group!("Benchmark results");
         for result in response.run.results {
             let benchmark_name = result.benchmark.name;
-            let time: humantime::Duration = Duration::from_secs_f64(result.time).into();
-            let time_text = style(format!("{}", time)).bold();
+            let time = helpers::format_duration(result.time, Some(2));
+
+            info!("{} - Time: {}", benchmark_name, style(time).bold());
```

Comment:
```suggestion
            info!("{}: {}", benchmark_name, style(time).bold());
```
this would be a bit clearer